.editorconfig

```diff
@@ -7,15 +7,5 @@ charset = utf-8
 indent_style = tab
 trim_trailing_whitespace = true
 
-[{*.yml,*.yaml}]
+[*.{yml,yaml}]
 indent_style = space
-indent_size = 2
-
-[package.json]
-indent_size = 2
```

Comment:
I added this initially to keep the visual style of `package.json` files the same since I thought they probably is a reason for it. Any insight here?